common-utils/src/common/deployment/__init__.py

```diff
@@ -213,40 +202,22 @@ def push_image(self, account_id: str) -> str:
 class ProjectEnvs(BaseModel):
     """Model that represents the project environment.
     This will store the list of docker envs to build and push.
+    The pyproject.toml configuration is used here.
     """
 
     docker_envs: list[FlowDockerEnv]
+    pyproject_metadata: PyProjectMetadata
```

Comment:
`ProjectEnvs` getting its pyproject.toml metadata from the caller now.  This is fine since only the internals of this module create these models.